### DISCUSSION TITLE
Auto Download new Episodes

### DISCUSSION THREAD

**Discussion by xTiLTEDx** [madeofpendletonwool/PinePods#162] [Ideas]
Hey madeofpendletonwool, 

can we get an optional Option to auto-Download Episodes (and new Episodes) automatically?

Thank you for your Hard work and the nice App!

**Comment by madeofpendletonwool** [comment]
That feature is on the way currently! I'm currently interested the process of fleshing out the download area and everything surrounding it in order to make archival really easy. This will includes nice to haves like auto downloads as you're describing. Keep an eye out for the next update coming in the next week or so!
